api/app/models/theq/timezone.py

```diff
@@ -0,0 +1,30 @@
+'''Copyright 2018 Province of British Columbia
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.'''
+
+from qsystem import db
+from app.models.theq import Base
+
+
+class Timezone(Base):
+
+    timezone_id = db.Column(db.Integer, primary_key=True, autoincrement=True, nullable=False)
+    timezone_abbreviation = db.Column(db.String(3), nullable=False)
```

Comment:
I think with the change on line 23, this also become superfluous.

Reply:
Again, thought I should be including all timezone information with a given zone, rather than omit it. We can choose to use it or not.

Reply:
What is the abbreviation for `America/Vancouver`?